src/components/footer.js

```diff
@@ -67,21 +67,77 @@ const CustomWrap = styled.div`
   text-align: center;
 `;
 
-const Footer = () => (
-  <FooterWrap>
-    <div className={wrap} />
-    <CustomWrap />
-    <div className={wrap}>
-      <Dropdown>
-        <div className={dropdownContent}>
-          <Element>1</Element>
-          <Element>2</Element>
-          <Element>3</Element>
+class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentLanguage: 'en',
+    };
+  }
+
+  changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
+    this.setState({ currentLanguage: lng });
+  };
+
+  chooseFlag = (lng) => {
```

Comment:
bardziej chyba powinno to być nazwane jako getFlag?
Pobierasz flagę tu, a nie wybierasz.

Reply:
zrobione